### DISCUSSION TITLE
(easy) way to disable to disable the whitelist?

### DISCUSSION THREAD

**Discussion by Cabbose2552** [uBlockOrigin/uBlock-issues#2070] [Ideas] [answered]
I would love to see the trusted sites whitelist be fully editable so i can discard protected elements if I so choose to do so. Iv found it very frustrating to try and block unwanted elements on my browser homepage (opera gx)  and in my extension settings pages. for example "super dark mode" has a sponsor spot that eats half the screen that I would love to do away with.

**Comment by ghost** [comment, marked as answer]
This is browser limitation. Trusted sites list is fully editable.
